src/ui/i18n.ts

```diff
@@ -1,20 +1,25 @@
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
-import Backend from 'i18next-http-backend';
+import * as resourcesToBackend from 'i18next-resources-to-backend';
 import { I18N_NAME_SPACE, KEEPERWALLET_DEBUG } from './appConfig';
 
+const backend = (
+  resourcesToBackend as unknown as typeof resourcesToBackend.default
+)((lng, ns, clb) => {
+  import(`../copied/_locales/${lng}/${ns}.${lng}.json`)
```

Comment:
I think we also need `webpackMode: 'eager'` magic comment to avoid creating and loading a separate file with translations:
https://webpack.js.org/api/module-methods/#magic-comments